sympy/series/approximants.py

```diff
@@ -101,3 +104,146 @@ def approximants(l, X=Symbol('x'), simplify=False):
         else:
             yield out
     return
+
+
+@public
+def pade_approximants(
+    f: Expr, x: Expr, order: int
+) -> Generator[tuple[Poly, Poly], None, None]:
```

Comment:
There needs to be `from __future__ import annotations` at the top of the file for this kind of annotation to work in older Python versions.